File: mobile/src/main/java/com/pdyjak/powerampwear/onboarding/AlmostReadyFragment.java
package com.pdyjak.powerampwear.onboarding;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.pdyjak.powerampwear.R;

public class AlmostReadyFragment extends Fragment {
    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
            @Nullable Bundle savedInstanceState) {
        return inflater.inflate(R.layout.onboarding_almost_ready, container, false);
    }
}

File: mobile/src/main/java/com/pdyjak/powerampwear/onboarding/AlmostReadyFragment.kt
package com.pdyjak.powerampwear.onboarding

import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import com.pdyjak.powerampwear.R

class AlmostReadyFragment : Fragment() {
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.onboarding_almost_ready, container, false)
    }
}
